src/main/java/org/folio/dew/batch/acquisitions/edifact/jobs/ExportHistoryTasklet.java

```diff
@@ -30,26 +32,40 @@ public class ExportHistoryTasklet implements Tasklet {
 
   private final KafkaService kafkaService;
   private final ObjectMapper objectMapper;
+  private final OrganizationsService organizationsService;
+
   @Value("#{jobParameters['jobId']}")
   private String jobId;
   @Override
-  public RepeatStatus execute(StepContribution contribution, ChunkContext chunkContext) {
+  public RepeatStatus execute(StepContribution contribution, ChunkContext chunkContext) throws Exception {
     var exportHistory = buildExportHistory(chunkContext);
 
     kafkaService.send(KafkaService.Topic.EXPORT_HISTORY_CREATE, null, exportHistory);
 
     return RepeatStatus.FINISHED;
   }
 
-  ExportHistory buildExportHistory(ChunkContext chunkContext) {
+  ExportHistory buildExportHistory(ChunkContext chunkContext) throws Exception {
+    var jobParameters = chunkContext.getStepContext().getJobParameters();
+    var ediExportConfig = objectMapper.readValue((String)jobParameters.get("edifactOrdersExport"), VendorEdiOrdersExportConfig.class);
```

Comment:
As I remember for these values "edifactOrdersExport", "edifactFileName" declared constans, so we can use these constants

Reply:
fixed